docs/source/notebooks/finch-usage.ipynb

```diff
@@ -98,155 +118,34 @@
-       "    table_id:                 Table A2 (17 November 2004)\n",
-       "    calendar:                 360_day\n",
-       "    project_id:               IPCC Fourth Assessment\n",
-       "    Conventions:              CF-1.0\n",
-       "    id:                       pcmdi.ipcc4.miub_echo_g.sresa2.run1.atm.da\n",
-       "    history:                  Mon Aug  1 11:43:58 2011: ncks -4 -L 7 -d lat,4...\n",
-       "    NCO:                      4.0.9\n",
-       "    climateindex_package_id:  https://github.com/Ouranosinc/xclim\n",
-       "    product:                  derived climate index\n",
-       "    institute_id:             CCCS</pre>"
-      ],
-      "text/plain": [
-       "<xarray.Dataset>\n",
-       "Dimensions:     (lat: 6, lon: 7, time: 20)\n",
-       "Coordinates:\n",
-       "    height      float64 ...\n",
-       "  * time        (time) object 2046-01-01 00:00:00 ... 2065-01-01 00:00:00\n",
-       "  * lat         (lat) float64 42.68 46.39 50.1 53.81 57.52 61.23\n",
-       "  * lon         (lon) float64 281.2 285.0 288.8 292.5 296.2 300.0 303.8\n",
-       "Data variables:\n",
-       "    frost_days  (time, lat, lon) timedelta64[ns] ...\n",
-       "Attributes:\n",
-       "    comment:                  Spinup: restart files from end of experiment 20...\n",
-       "    title:                    MIUB  model output prepared for IPCC Fourth Ass...\n",
-       "    cmor_version:             0.96\n",
-       "    institution:              Canadian Centre for Climate Services (CCCS)\n",
-       "    source:                   ECHO-G(1999): atmosphere: ECHAM4 (T30L19) with ...\n",
-       "    contact:                  Canadian Centre for Climate Services\n",
-       "    references:               ECHAM4: E. Roeckner et al., 1996, The atmospher...\n",
-       "    experiment_id:            SRES A2 experiment\n",
-       "    realization:              1\n",
-       "    directory:                /ipcc/sresa2/atm/da/\n",
-       "    table_id:                 Table A2 (17 November 2004)\n",
-       "    calendar:                 360_day\n",
-       "    project_id:               IPCC Fourth Assessment\n",
-       "    Conventions:              CF-1.0\n",
-       "    id:                       pcmdi.ipcc4.miub_echo_g.sresa2.run1.atm.da\n",
-       "    history:                  Mon Aug  1 11:43:58 2011: ncks -4 -L 7 -d lat,4...\n",
-       "    NCO:                      4.0.9\n",
-       "    climateindex_package_id:  https://github.com/Ouranosinc/xclim\n",
-       "    product:                  derived climate index\n",
-       "    institute_id:             CCCS"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Metalink content-type detected.\n",
+      "Downloading to /tmp/tmp8fi43lm1/frost-days_SRES-A2-experiment_20460101-20650101.nc.\n"
```

Comment:
This random path most likely will break Jenkins.  But I don't have an updated Finch server to test.  Will do a blind `output-sanitize.cfg` fix.